src/main/app/claims/models/claim.ts

```diff
@@ -646,4 +646,8 @@ export class Claim {
   private hasOrderBeenDrawn (): boolean {
     return !!this.directionOrder
   }
+
+  public isIntentionToProceedEligible (): boolean {
```

Comment:
Do you need method to be public?

Reply:
it is being called from the nunjucks file. I was just emulating one of the other calls